src/Module/Setup.php

```diff
@@ -0,0 +1,326 @@
+<?php
+/**
+ * @file src/Core/System.php
+ */
+namespace Friendica\Module;

```

Comment:
Modules are special classes intended to be called as `https://friendica.example.com/module`. Look at the `BaseModule` class. If your class doesn't need to extend `BaseModule`, then it probably shouldn't be in `Friendica\Module`. May I suggest moving it to `Friendica\Core` instead?

Reply:
I created the class at Core first, but it seemed for me that they are all doesn't have any private methods and so I guessed that that they are for public methods only ;-)

I'll move it to Core

Reply:
If you move functions to classes, can you please transfer the method names to camelCase (e.g. `private static function checkPhp(&$phpath, &$checks)`).

Reply:
@rabuzarus I would call this checkPHP, since PHP is an abbreviation.